core/src/apps/cardano/layout.py

```diff
@@ -568,6 +620,23 @@ async def confirm_withdrawal(
     )
 
 
+def _format_stake_credential(
+    path: list[int], script_hash: bytes | None, key_hash: bytes | None
+) -> tuple[str, str]:
+    if path:
+        return (
+            f"for account {format_account_number(path)}:",
+            address_n_to_str(to_account_path(path)),
+        )
+    elif key_hash:
+        return ("for key hash:", format_key_hash(key_hash, False))
```

Comment:
These key hashes should actually all be staking key hashes and thus we should display them as such i.e. with the `stake_vkh` prefix --> https://github.com/cardano-foundation/CIPs/tree/master/CIP-0005#hashes. I suggest updating the `format_key_hash` to take another boolean flag (e.g. `is_staking_key`) and decide based on that.

Reply:
Fixed in https://github.com/vacuumlabs/trezor-firmware/pull/71/commits/700d4ad00e199c1515de41160b4c3b25d5b59fda, except for the required signer displaying.